onnx/version_converter/adapters/NoPreviousVersionAdapter.h

```diff
@@ -0,0 +1,19 @@
+// Adapter indicating lack of a previous version of some op before a given
+// opset version.
+
+#pragma once
+
+#include "onnx/version_converter/adapters/adapter.h"
+
+namespace ONNX_NAMESPACE { namespace version_conversion {
+
+struct NoPreviousVersionAdapter final : public Adapter {
+  explicit NoPreviousVersionAdapter(const std::string& op_name, const OpSetID&
```

Comment:
You can probably move in `op_name`, `initial` and `target`.

Reply:
I also need to do this for BackwardsCompatibleAdapter